<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-KJQ7RLT" failures="1" tests="3" name="Test" time="21.269" errors="0" timestamp="2025-10-20T22:13:53 IST">
  <testcase classname="day46.OrangeHRMDemo" name="TestImage" time="0.047"/>
  <testcase classname="day46.OrangeHRMDemo" name="testURL" time="0.022">
    <failure type="java.lang.AssertionError" message="expected [https://opensource-demo.orangehrmlive.com/w] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]">
      <![CDATA[java.lang.AssertionError: expected [https://opensource-demo.orangehrmlive.com/w] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]
at day46.OrangeHRMDemo.testURL(OrangeHRMDemo.java:40)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- testURL -->
  <testcase classname="day46.OrangeHRMDemo" name="testApptitle" time="0.0">
    <skipped/>
  </testcase> <!-- testApptitle -->
</testsuite> <!-- Test -->
